index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
```

Comment:
please delete the free space

Reply:
[OPTIONAL] - It would be better to explain the `why` aspect with some reference when you perform a code review; kindly use descriptive, polite, and clear communication.


Example

- [ ] Kindly remove the free space between HTML line code to be your code clean and professional. For further details please check [this link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)